src/main/java/com/github/maven_nar/AbstractCompileMojo.java

```diff
@@ -121,6 +121,16 @@
   @Parameter
   private Java java;
 
+    /*
```

Comment:
Note that this is not javadoc, but should be. And note that javadoc is HTML, so line breaks mean nothing without `<p>` or `<br>` tags. Best is to use the IDE to preview the final result.